### ISSUE TITLE
Let's encrypt certificates expiring 9 april

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-devops#75]
Your certificate (or certificates) for the names listed below will expire in
9 days (on 09 Apr 17 23:04 +0000). Please make sure to renew
your certificate before then, or visitors to your website will encounter errors.

- scigraph-ontology.monarchinitiative.org
- scigraph-data.monarchinitiative.org
- genome.monarchinitiative.org

For any questions or support, please visit https://community.letsencrypt.org/.
Unfortunately, we can't provide support by email.

For details about when we send these emails, please visit
https://letsencrypt.org/docs/expiration-emails/. In particular, note
that this reminder email is still sent if you've obtained a slightly
different certificate by adding or removing names. If you've replaced
this certificate with a newer one that covers more or fewer names than
the list above, you may be able to ignore this message.

**Comment by vanmatre** [comment]
I'm poking around and do not see any installed certs or tools to install the certs.

QuestionS: Who was working on the certs and installed them? were these certs generated and never installed?

Inquiring minds want to know....

**Comment by jnguyenx** [comment]
I installed and maintain those. I'll look into it, thanks!

**Comment by vanmatre** [comment]
Certbot can do auto-renew if that can work for you....

**Comment by TomConlin** [comment]
Ubuntu 16.04 LTS certbot  may need tweaking they shipped an old agent

https://community.letsencrypt.org/t/solved-problem-renewing-certificate-with-certbot-ubuntu-16-04-apache-2-4-18/23813

**Comment by jnguyenx** [comment]
Yeah I had issues with certbot before as mentioned in another issue https://github.com/monarch-initiative/monarch-devops/issues/71#issuecomment-284929964.

However I've just noticed that they have they own ppa now, I'll give it a shot!

**Comment by jnguyenx** [comment]
I switched to Certbot to renew the certificate. In most of the cases it worked well: for a basic nginx configuration it went smoothly, but for more complex ones (as for monarch-ttl-data), it generated an invalid nginx conf, I manually had to fix it.

Some certs will expired on **June 10th**, @jmcmurry can you monitor the email box around **June 1st**, if we don't receive any notifications it means that renewal works as expected.

**Comment by jmcmurry** [comment]
sure; sounds good thanks

**Comment by jmcmurry** [comment]
Hello,

Your certificate (or certificates) for the names listed below will expire in
19 days (on 17 May 17 21:48 +0000). Please make sure to renew
your certificate before then, or visitors to your website will encounter errors.

archive.monarchinitiative.org

For any questions or support, please visit https://community.letsencrypt.org/.
Unfortunately, we can't provide support by email.

For details about when we send these emails, please visit
https://letsencrypt.org/docs/expiration-emails/. In particular, note
that this reminder email is still sent if you've obtained a slightly
different certificate by adding or removing names. If you've replaced
this certificate with a newer one that covers more or fewer names than
the list above, you may be able to ignore this message.

If you want to stop receiving all email from this address, click
http://mandrillapp.com/track/unsub.php?u=30850198&id=8d59565b40484c10a6990c2515add343.SEDj0DnRMKZe3RZuczzjOxHI63M%3D&r=https%3A%2F%2Fmandrillapp.com%2Funsub%3Fmd_email%3Dinfo%2540monarchinitiative.org
(Warning: this is a one-click action that cannot be undone)

Regards,
The Let's Encrypt Team

**Comment by TomConlin** [comment]
@jmcmurry reopen and assign me please

**Comment by TomConlin** [comment]
/boot on archive  is out of disk space

    /dev/vda1                     472M  439M  8.7M  99% /boot

cleaning out old  kernels  

```sudo apt-get autoremove```  

    /dev/vda1                     472M  152M  296M  34% /boot

**Comment by TomConlin** [comment]
after making room on the root partition the monthly cron  succeeds  

   sudo -H /etc/cron.monthly/letsencrypt_renew.sh 
    ...
   - Congratulations! Your certificate and chain have been saved at
   /etc/letsencrypt/live/archive.monarchinitiative.org/fullchain.pem.
   Your cert will expire on 2017-07-27.

**Comment by vanmatre** [comment]
Great catch,

 I usually run into this with updates. Autoremove tends to take out all the older kernels but I like to keep at least the last working kernel or 2 around....

There must be a switch to specify how many to keep.


--Ethan


Ethan VanMatre
Informatics Research Analyst
Institute on Development and Disability
Oregon Health & Science University
CSLU - GH40
3181 SW Sam Jackson Park Rd
Portland, OR 97239
(503) 346-3764
vanmatre@ohsu.edu
________________________________
